### DISCUSSION TITLE
Pキーが反応しない

### DISCUSSION THREAD

**Discussion by YHirokiY** [Yowkees/keyball#311] [Q&A]
![Untitled (Draft)-2](https://github.com/Yowkees/keyball/assets/106242937/483e8b14-44b9-498b-9d71-ad9a6b1a947c)

それ以外のキーは全て反応するのですが、赤丸のキー（右側Pキー）が反応しないです。

解決方法を教えていただけませんか

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

下記画像の緑丸で示したダイオードがPキーに対応するダイオードです。この半田付けや極性をご確認ください。

また、PCBソケットの取り外し時等に基板の配線がダメージを受けた可能性もありますので、下記画像のオレンジと水色の同色どうしが導通しているかご確認お願いします。
もし断線している組み合わせがあれば、エナメル線やゴム被覆導線で接続していただければ修復できると思います。

お手数ですがご対応お願いいたします。
![KB39_Pfix](https://github.com/Yowkees/keyball/assets/84209772/7ca09f0c-cb20-4d19-99f8-8f9aa111e1c4)

**Reply by YHirokiY** [reply]
どうやらダイオードの部分が原因だったようで、ダイオードを取り替えたら解決しました！

迅速に対応していただきありがとうございます！

**Comment by Yowkees** [comment]
ダイオードの問題でしたか。解決できて安心しました。
今後ともご愛用いただければ幸いです。
